web/src/store/index.ts

```diff
@@ -114,6 +128,20 @@ export const actions = actionTree(
         // localStorage.removeItem('tokenExpiration');
       }
     },
+    setProfile(vuexContext, profile: Profile) {
+      return this.$axios
+        .$post('/user-profile', {
```

Comment:
Did you wanna take a shot at adding this API route or should I add it? We could always just return back what we send to the API with a `200` since we don't have a DB yet.

Reply:
I started messing with it! I can push up what I have. I'm having a hard time with the store. I don't think that accessing store.getters in a component is the way to go.